docs/GoGetSllDoc.md

```diff
@@ -0,0 +1,669 @@
+            "filename": "",
+            "content": ""
+        }
+    },
+    "order_id": "",
+    "invoice_id": "",
+    "order_status": "",
+    "success": "",
+    "san": [
+        {
+            "san_name": "",
+            "validation_method": "",
+            "status": 1,
+            "status_description": "",
+            "validation": {
+                "http": {
+                    "link": "",
+                    "filename": "",
+                    "content": ""
+                }
+            }
+        }
+    ],
+    "order_amount": "",
+    "currency": "",
+    "tax": "",
+    "tax_rate": ""
+}</pre>
+
+<h3>operation: Reissue SSL Order</h3>
+
+<h4>Input parameters</h4>
+
+<table border=1><thead><tr><th>Parameter</th><th>Description</th></tr></thead><tbody><tr><td>Order ID</td><td>Specify the ID of the order that uniquely identifies the SSL certificate associated with the order you wish to reissue.</td></tr>
+<tr><td>Certificate Signing Request (CSR)</td><td>Specify the Certificate Signing Request (CSR) is a base64-encoded string that contains information required to create a new SSL certificate.</td></tr>
+<tr><td>Web Server Type</td><td>Specify the type of web server that the reissued SSL certificate.</td></tr>
+<tr><td>DCV Method</td><td>Specify the method used to validate domain control as part of the SSL certificate reissue process</td></tr>
+<tr><td>DNS Names</td><td>Specify the CSV list of domain names or subdomains that you want to include in the reissued SSL certificate.</td></tr>
+<tr><td>Approver Emails</td><td>(Optional) Specifies the email addresses of the individuals who are authorized to approve the reissue of the SSL certificate.</td></tr>
+<tr><td>Approver Email</td><td>(Optional) Specifies the email address of the individual who will receive the approval request for the SSL certificate reissue.</td></tr>
+<tr><td>Signature Hash</td><td>(Optional) Specifies the hashing algorithm used to sign the Certificate Signing Request (CSR). </td></tr>
+<tr><td>Additional Field</td><td>(Optional) Specify any additional fields required for the use case, and provide them in JSON format.</td></tr>
+</tbody></table>
+
+<h4>Output</h4>
+
+<p>The output contains the following populated JSON schema:</p>
+
+<pre>{
+    "product_id": "",
+    "additional_domains_count": "",
+    "additional_domains_price": "",
+    "approvalEmails": [],
+    "validation": {
+        "dns": {
+            "record": ""
+        }
+    },
+    "order_id": "",
+    "order_status": "",
+    "success": ""
+}</pre>
+
+<h3>operation: Add SSL Renew Order</h3>
+
+<h4>Input parameters</h4>
+
+<table border=1><thead><tr><th>Parameter</th><th>Description</th></tr></thead><tbody><tr><td>Product ID</td><td>Specify the product ID for which the SSL certificate is being renewed.</td></tr>
+<tr><td>Certificate Signing Request (CSR)</td><td>Specify the CSR (Certificate Signing Request) encoded text for the certificate you want to renew.</td></tr>
```

Comment:
Update the description:

e.g. Specify the CSR (Certificate Signing Request) for the certificate you want to renew.